### PR TITLE
[CI] Azp bug fix

### PR DESCRIPTION
**Pull request by liushilongbuaa**

### CODE REVIEW

**Comment by lguohan** [comment]
can you use sonic-snmpagent as example?

**Comment by liushilongbuaa** [comment]
> can you use sonic-snmpagent as example?

I used snmpagent as example in this commit 'd34050d' using sonic-slave-buster image and 'python3 setup.py test' for unit test.
But there are no test result files created.

File: azure-pipelines.yml [hunk 1]
```diff
@@ -21,19 +19,34 @@ stages:
   - job:
     displayName: "build"
     timeoutInMinutes: 60
+    pool:
+      vmImage: ubuntu-20.04
```
Comment (lguohan) [inline azure-pipelines.yml:23]: use container please

File: azure-pipelines.yml [hunk 2]
```diff
@@ -21,19 +19,34 @@ stages:
...
+        python3 setup.py bdist_wheel
+        mkdir -p target/python-wheels/
+        cp dist/*.whl $(Build.ArtifactStagingDirectory)/
+      displayName: "Build"
+    - script: |
+        set -ex
+
+        sudo pip3 install pytest==4.6.2 attrs==19.1.0 exabgp==4.0.10 distro==1.5.0 docker==4.4.1 redis==3.3.4 flaky==3.7.0
+        sudo py.test -v --junitxml=tr.xml
+      displayName: "Unit tests"
```
Comment (lguohan) [inline azure-pipelines.yml:42]: can you refer to example in sonic-snmpagent?

**Review by lguohan — CHANGES_REQUESTED** [review]
as comments

File: azure-pipelines.yml (hunk 2 above)
Comment (lguohan) [reply azure-pipelines.yml:42]: please add pytest.ini properly

**Pull request closed without merging**